charts/atlantis/templates/statefulset.yaml

```diff
@@ -128,12 +137,21 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          {{- if or .Values.gitconfig .Values.gitconfigSecretName }}
+          {{- if or .Values.gitconfig .Values.gitconfigSecretName .Values.netrc .Values.netrcSecretName }}
           lifecycle:
             postStart:
               exec:
-                command: ["/bin/sh", "-c", "cp /etc/secret-gitconfig/gitconfig /home/atlantis/.gitconfig && chown atlantis /home/atlantis/.gitconfig"]
-          {{- end}}
+                command:
+                  - "/bin/sh"
+                  - "-c"
+                  {{- if and (or .Values.gitconfig .Values.gitconfigSecretName) (or .Values.netrc .Values.netrcSecretName) }}
+                  - "cp /etc/secret-gitconfig/gitconfig /home/atlantis/.gitconfig && chown atlantis /home/atlantis/.gitconfig && cp /etc/secret-netrc/netrc /home/atlantis/.netrc && chown atlantis /home/atlantis/.netrc"
+                  {{- else if or .Values.gitconfig .Values.gitconfigSecretName }}
+                  - "cp /etc/secret-gitconfig/gitconfig /home/atlantis/.gitconfig && chown atlantis /home/atlantis/.gitconfig"
+                  {{- else if or .Values.netrc .Values.netrcSecretName }}
+                  - "cp /etc/secret-netrc/netrc /home/atlantis/.netrc && chown atlantis /home/atlantis/.netrc"
+                  {{- end }}
```

Comment:
```suggestion
                  {{- if or .Values.gitconfig .Values.gitconfigSecretName }}
                  - "cp /etc/secret-gitconfig/gitconfig /home/atlantis/.gitconfig && chown atlantis /home/atlantis/.gitconfig"
                  {{- end }}
                  {{- if or .Values.netrc .Values.netrcSecretName }}
                  - "cp /etc/secret-netrc/netrc /home/atlantis/.netrc && chown atlantis /home/atlantis/.netrc"
                  {{- end }}
```

Reply:
If a user sets both the gitconfig and netrc then the suggested change won't work. It will only execute the first command. For example it would be the equivalent of running the following and only the first output being executed:
```
$ /bin/sh -c "echo 'test'" "echo 'test2"
test
```

Reply:
The suggested change says that if you set gitconfig, write it. If you set netrc, write it. If you set both, it will write both.

With your logic, it says, if you use both, write both, if you use gitconfig, write only gitconfig. If you use netrc, write only netrc.

Both the logic results in the same but i thought mine was simpler. Am i wrong here?

Reply:
cc @jamengual @chenrui333 for an extra set of eyes

Reply:
The logic is the same, but the syntax of two cp commands as separate items in the list results in only the first cp command being executed. However, I think it’s best to get rid of this block all together and just do the direct mount